scripts/test_execution_tutorial.sh

```diff
@@ -0,0 +1,20 @@
+set -e # first error should stop execution of this script
+
+# Identify the bin dir in the distribution, and source the common include script
+BASE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )"/.. && pwd )"
+source ${BASE_DIR}/scripts/common.sh
+
+# also trap errors, to reenable terminal settings
+trap onExit ERR
+export CLASSPATH=tutorial/execution-tutorial/target/execution-tutorial-assembly-0.13.1.jar
+time $SCALD \
```

Comment:
I was working with @ianoc on this. Previously what I put here is "scala -classpath ...", but travis complains it couldn't find "scala" command. So we figured out travis uses scald.rb to setup scala version etc. Not sure if java -jar will directly work. I can try.

One side question, @ianoc and I find that the onComplete doesn't always guarantee to run since it waits a Future. The submitter node's jvm can exit quietly before the results are ready. We're wondering should we consider developing an inline side-effect like foreach to Execution?